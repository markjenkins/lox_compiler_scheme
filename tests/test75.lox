// from craftinginterpreters/test/precedence.lox

// 1 - 1 is not space-sensitive.
print 1 - 1; // expect: 0
print 1 -1;  // expect: 0
print 1- 1;  // expect: 0
print 1-1;   // expect: 0
