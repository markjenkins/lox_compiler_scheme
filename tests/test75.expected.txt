OP_CONSTANT 1
OP_CONSTANT 1
OP_SUBTRACT
OP_PRINT
OP_CONSTANT 1
OP_CONSTANT 1
OP_SUBTRACT
OP_PRINT
OP_CONSTANT 1
OP_CONSTANT 1
OP_SUBTRACT
OP_PRINT
OP_CONSTANT 1
OP_CONSTANT 1
OP_SUBTRACT
OP_PRINT
OP_RETURN
